eslint.config.mjs

```diff
@@ -0,0 +1,366 @@
+import jest from 'eslint-plugin-jest';
+import lwcInternal from '@lwc/eslint-plugin-lwc-internal';
+import _import from 'eslint-plugin-import';
+import header from 'eslint-plugin-header';
+import { fixupPluginRules } from '@eslint/compat';
+import globals from 'globals';
+import js from '@eslint/js';
+import tseslint from 'typescript-eslint';
+
+export default [
+    {
+        ignores: [
+            '**/node_modules/',
+            '**/dist/',
+            '**/lib/',
+            '**/coverage/',
+            '**/fixtures/',
+            '**/public/',
+            '**/__benchmarks_results__/',
+            '**/playground/',
+            '**/*.html',
+            '**/*.css',
```

Comment:
```suggestion
```
I don't think eslint can handle these, so I don't think we need to ignore them.

Reply:
This was being covered by the `header/header` related rule. Since that whole rule was nuked, I am going to remove that section.